tests/test_peer_did.py

```diff
@@ -1,13 +1,108 @@
 import pytest
 
```

Comment:
It makes sense to split the tests into multiple files: `test_peer_did_creation`, `test_peer_did_resolving`, `test_peer_did_saving`.